app.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topics.controller");
+const { handleCustomError } = require("./errors/custom-errors");
+const { getEndpoints } = require("./controllers/endpoints.controller");
+const app = express();
+
+app.get("/api", getEndpoints);
+
+app.get("/api/topics", getTopics);
```

Comment:
Yes, good implementation of the 2 routes here and pointing to controller functions.